index.html

```diff
@@ -329,6 +326,42 @@ <h2 class="top-heading">About &nbsp; &nbsp; &nbsp; &nbsp; Myself</h2>
           <img class="sep1" src="./images/Separator Bottom.png" alt="line" />
         </div>
       </section>
+      <section class="Contact-me">
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/{form_id}" methode="post">
```

Comment:
- [ ] It looks like you have not added your Formspree account ID in the placeholder (where there is` {form_id}`). Please create an account on [Formspree](https://formspree.io/html/) and then you will be given an ID which you will add here in your form attribute. This helps to submit all form data into your account on Formspree 😃 👍